### PR TITLE
Add konnector version in job execution message & improve Service logs

### PR DESCRIPTION
**Pull request by maximemenager**

### CODE REVIEW

File: pkg/workers/exec/konnector.go [hunk 1]
```diff
@@ -55,6 +55,7 @@ type KonnectorMessage struct {
 	FolderToSave   string `json:"folder_to_save"` // FolderToSave is the identifier of the folder
 	AccountDeleted bool   `json:"account_deleted,omitempty"`
 
+	Version string `json:"version"`
```
Comment (nono) [inline pkg/workers/exec/konnector.go]: I think it should be removed (just a remaining line of code from a reverted commit), no?

File: pkg/workers/exec/service.go [hunk 2]
```diff
@@ -50,6 +50,9 @@ func (w *serviceWorker) PrepareWorkDir(ctx *jobs.WorkerContext, i *instance.Inst
 		}
 		return
 	}
+
+	w.slug = slug
```
Comment (nono) [inline pkg/workers/exec/service.go]: By curiosity, why did you move this assignation?

Comment (maximemenager) [reply pkg/workers/exec/service.go]: Only for the logs. If we get the manifest but have any kind of error after (app not ready, service not found, ....), we will still have the slug name instead of an empty one

File: pkg/workers/exec/konnector.go (hunk 1 above)
Comment (maximemenager) [reply pkg/workers/exec/konnector.go]: Yep, nice find ;)
